package.json

```diff
@@ -81,7 +81,6 @@
   },
   "lint-staged": {
     "src/**/*.{ts,tsx}": [
-      "yarn lint:fix",
```

Comment:
what is the reason for remove this?

Reply:
it was removed by mistake. added in the commit below.